bower.json

```diff
@@ -6,34 +6,34 @@
     "build/ng-admin.min.js"
   ],
   "dependencies": {
-    "angular": "~1.3.1",
+    "angular": "~1.3.1"
+  },
+  "devDependencies": {
```

Comment:
It's the full the default one, I've named it `ng-admin-only.min.js` to not break compatibility.